tyr/lifecycle/asgevent.py

```diff
@@ -0,0 +1,38 @@
+import boto3
+import logging
+
+class ASGEvent(object):
+    def __init__(self, hook_name=None,
+                asg_name=None,
+                transition='autoscaling:EC2_INSTANCE_LAUNCHING',
+                role_name=None):
+
+        logging.getLogger('botocore').setLevel(logging.CRITICAL)
+        self.log = logging.getLogger('Tyr.Lifecycle.ASGEvent')
```

Comment:
Take a look at [this][0].

[0]: https://github.com/hudl/Tyr/blob/master/tyr/servers/server.py#L77